SSA/Experimental/Bits/Fast/BitStream.lean

```diff
@@ -367,28 +367,63 @@ variable {w : Nat} {x y : BitVec w} {a b a' b' : BitStream}
 
 local infix:20 " ≈ʷ " => EqualUpTo w
 
-theorem xor_xor_eq_not {a b : Bool} : xor (!xor a b) b = !a := by
-  cases a
-  <;> cases b
-  <;> simp
-
-theorem xor_and_eq_and {a b : Bool} : (!xor a b && b) = (a && b) := by
-  cases a
-  <;> cases b
-  <;> simp
-
```

Comment:
I would suggest you move these statements into ForLean in a separate PR (no review required).

Reply:
... such that you can then rebase this PR on top of the cleanup, which reduces the size of this PR.

Reply:
I implemented this.